microsoft-365/managed-desktop/service-description/DeviceRegistration.md

```diff
@@ -10,52 +10,44 @@ ms.localizationpriority: medium
 # Register devices in Microsoft Managed Desktop
 
 [//]: # (This draft version topic focuses on the self-service option--how different is the partner thing?)
->Note: This article describes the steps for you to register devices on your own. The process for Partners is documented [Somewhere else LINK].
+
+
+>[!NOTE]
+>This topic focuses describes the steps for you to register devices on your own. If you're working with a Partner, see [Somewhere else LINK].
 
 Managed Desktop can work with brand-new devices or you re-use devices you might already have (which will re-image them). You can register devices by using the Managed Desktop Admin Center or save time and gain flexibility by using an API.
 
+
 ## Prepare to register devices
 
 If you haven't already obtained the devices you want to use, see [Order devices LINK] for options to get some. In either case, you should check {device list LINK} to ensure that they meet the appropriate system requirements.
 
 Whether you're working with completely new devices or re-using existing ones, to register them with Microsoft Managed Desktop, you'll need to prepare a comma-delimited (CSV) file. This file should include the following information for each device:
 
->Note: This format is for self-service only. Partners have a different format documented [Somewhere else LINK].
+>[!NOTE]
+>You should use this format only if you are registering the devices yourself. If you're working with a Partner, see [Somewhere else LINK].
 
 [//]: # (I see a mention in the Word outline of this CSV file being different depending on the "actor"--can you clarify?)
 [//]: # (Pretty different - the fields for Manufacturer, Model, and Serial have to be EXACT matches, and Hardware Hash is not required at all.)
 
-These values are used for display purposes, and don't need to match properties from the device exactly.
+These values are used for display purposes, and don't need to match properties from the device exactly. For example, it doesn't matter if you list a device model as "Surface" or "Surface Laptop."
+
 - Device manufacturer (example: Microsoft) 
 - Device model (example: Surface Laptop)
 - Device serial number
-
-The Hardware hash must be an exact match.
 - Hardware hash
 
-To obtain the hardware hash you can ask for help from your OEM or Partner, or follow these steps for each device:
-
-1. Open an admin PowerShell window
-2. Install-Script -Name Get-WindowsAutoPilotInfo
-3. Set-ExecutionPolicy -ExecutionPolicy Unrestricted
-4. .\Get-WindowsAutoPilotInfo -OutputFile <path>\hardwarehash.csv
+>[!NOTE]
```

Comment:
Can we maybe put this note before the Hardware Hash? I'm worried folks scanning the list may miss this important piece.

Reply:
Sorry, which Note exactly? I mean probably yes, but I want to be sure I'm looking at the right one.

Reply:
The one that reads "Unlike the other values, the hardware hash..."